.github/workflows/deployment-report.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async ({ github, context }) => {
+  const sha = context.sha
+  const owner = context.payload.repository.owner.login
+  const repo = context.payload.repository.name
+  const { data: pulls } = await github.rest.pulls.list({
+    state: 'open',
+    owner,
+    repo
+  })
+  const actionPull = pulls.find(pull => pull.head.sha === sha)
+  return actionPull.number
+}
```

Comment:
The `deployment_status` event we use from Heroku doesn't actually include info about the pull request, so we have to duck into the GitHub API to get that PR via the current commit's `sha`.